_posts/2021-12-04-pdf-protection.adoc

```diff
@@ -0,0 +1,454 @@
+for copyright and content protection concerns (other than DRM).
+
+Sort of as a "`your eyes only`" document, where you can read but not copy out
+content.
+
+This permission can be disallowed using the following document attribute:
+
+* `:pdf-allow-copy-content: false` specifies that copying of content is
+disallowed.
+
+[example]
+====
+[source,adoc]
+----
+= Copy Pasta
+:pdf-allow-copy-content: false
+:pdf-owner-password: D0nt700k4tM3
+...
+----
+====
+//:pdf-allow-edit-content: false
+
+
+=== Disallow editing of content
+
+Editing of PDF content can be disallowed to ensure that the document content remains as when it was originally published.
+
+This permission can be disallowed using the following document attribute:
+
+* `:pdf-allow-assemble-document: false` specifies that editing of PDF content is disallowed.
+
+[example]
+====
+[source,adoc]
+----
+= No Edit
+:pdf-allow-edit-content: false
+:pdf-owner-password: D0nt700k4tM3
+...
+----
+====
+
+=== Disallow inserting, deleting or rotating pages
+
+A PDF file that is marked to disallow page modification means an unauthorized
+user cannot modify its content by inserting, deleting or rotating pages.
+
+This permission can be disallowed using the following document attribute:
+
+* `:pdf-allow-assemble-document: false` specifies that copying of content is
+disallowed.
+
+[example]
+====
+[source,adoc]
+----
+= No Page Shuffle
+:pdf-allow-assemble-document: false
+:pdf-owner-password: D0nt700k4tM3
+...
+----
+====
+
+
+=== Disallow adding annotations or signatures
+
+Annotations and signatures are content that can be added to a PDF file.
+A PDF file can be marked to prevent annotating and signatures.
+
+This permission can be disallowed using the following document attribute:
+
+* `:pdf-allow-edit-annotations: false` specifies that annotations and signatures cannot be added.
+
+[example]
+====
+[source,adoc]
+----
+= Copy Pasta
+:pdf-allow-edit-annotations: false
+:pdf-owner-password: D0nt700k4tM3
+...
+----
+====
+
+
+=== Disallow printing
+
+A PDF file marked "`not for printing`" means exactly that, the PDF content
+cannot be printed on a printer.
+
+Compliant PDF readers will block not only printing directly to a physical
+printer, they would also block printing to files (e.g. features like "`print to
+PDF`" or "`save as image`").
+
+This permission can be disallowed using the following document attribute:
+
+* `:pdf-allow-print: false` specifies that the PDF document cannot be printed.
+
+
+[example]
+====
+[source,adoc]
+----
+= Print Me, Will You?
+:pdf-allow-print: false
+:pdf-owner-password: D0nt700k4tM3
+...
+----
+====
+
+
+=== PDF protection against high quality printing
+
+For these PDFs, low quality prints are allowed, but not high resolution prints.
+The intent for this limitation is to prevent an accurate physical reproduction
+of documents that could potentially mislead someone that a print out (from this
+file) was authentic.
```

Comment:
"authentic" is not the word you want here. Authorised? Official?

Reply:
Maybe? I can go with "authorised", but with "authentic" I mean whether the content has been maliciously adjusted.